### ISSUE TITLE
Loading component stop working after using vue-router in page method

### ISSUE THREAD

**Issue by mkv27** [nuxt/nuxt.js#113]
When I try with `this.$nuxt.$router.replace({ path: '/different-layout-route'})` for browser path change to a different layout, the `progress div` stop working.

I don't know if $router in a method page is the right way to browser redirects.

Thanks for the amazing nuxtjs!

**Comment by Atinux** [comment]
Hi @mkv27 

Where to you use `this.$nuxt.$router.replace({ path: '/different-layout-route'})`?

What expected behaviour do you want to see when replacing the route?

Thank you for trying Nuxt.js!

**Comment by mkv27** [comment]
Hi @Atinux

I'm using the Facebook SDK for sign in users. 
I have a `pages/login.vue` component and I want to replace the url to a 'secret' dashboard  page after success FB login:

```js
export default {
  methods: {

    login () {
      FB.login( function( response ){
        if ( response.authResponse ){
            ...
            // replace /dashboard path
            self.$nuxt.$router.replace({ path: 'dashboard' })
        }
      )
    }
  
  }	
}
```

I want to see exactly the same behaviour as `<nuxt-link>` click.

**Comment by Atinux** [comment]
I see, indeed it's a bug @mkv27, good catch!

I'll work on it and it will be fixed in the next release!

**Comment by Atinux** [comment]
It's now fixed in the latest release [0.9.7](https://github.com/nuxt/nuxt.js/releases/tag/v0.9.7).
